packit_service/worker/handlers/github_handlers.py

```diff
@@ -204,12 +205,15 @@ def run(self) -> TaskResults:
                 err_without_new_lines = err.replace("\n", " ")
                 branch_errors += f"| `{branch}` | `{err_without_new_lines}` |\n"
 
+            msg_retrigger = MSG_RETRIGGER.format(
+                job="update", command="propose-downstream", place="issue"
+            )
             body_msg = (
                 f"Packit failed on creating pull-requests in dist-git:\n\n"
                 f"| dist-git branch | error |\n"
                 f"| --------------- | ----- |\n"
                 f"{branch_errors}\n\n"
-                f"{MSG_RETRIGGER.format(job='update', command='propose-update', place='issue')}\n"
+                f"{msg_retrigger}\n"
```

Comment:
I'm just wondering if we want to place a comment here if a user writes propose-update instead of propose-downstream

Reply:
I don't think so. The message already says to use `propose-downstream` so the user should realize the change anyway. Thanks for review.